python/morpheus/morpheus/_lib/include/morpheus/controllers/monitor_controller.hpp

```diff
@@ -114,11 +120,11 @@ class ProgressBarContextManager
         {
             pbar->print_progress(true);
             m_stdout_os << termcolor::reset;  // The font option only works for the current bar
-            m_stdout_os << "\n";
+            m_stdout_os << "\n" << std::flush;
```

Comment:
newline + flush == `std::endl`
```suggestion
            m_stdout_os << std::endl;
```